.github/workflows/verify-build.yml

```diff
@@ -0,0 +1,38 @@
+name: Verify Build for Vanilla Components
+
+on:
+  push:
+    branches:
+      - main
+
+jobs:
+  build:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-latest, macOS-latest, windows-latest]
+
+    steps:
+      - name: Test on ${{ matrix.os }}
+        run: echo "Testing on ${{ matrix.os }}"
+
+      - name: Checkout main branch
+        uses: actions/checkout@v2
+
+      - name: Setup Node.js
+        uses: actions/setup-node@v2
+        with:
+          node-version: 20.x
+
+      - name: Install dependencies
+        run: npm i
+
+      - name: Build embeddable
+        run: npm run embeddable:build
+
+      - name: Check for errors
```

Comment:
Is this step needed? Won't it automatically fail when some of previous command fails?

Reply:
Yes we can remove it